app/views/layouts/component_preview.html.erb

```diff
@@ -2,8 +2,9 @@
 <html>
   <head>
     <title>Component Preview</title>
+    <%= stylesheet_link_tag 'application', nopush: false %>
     <%= render_stylesheet_once_tags %>
-    <%= stylesheet_link_tag 'application', media: 'all' %>
+    <%= stylesheet_link_tag 'utilties', nopush: false %>
```

Comment:
```suggestion
    <%= stylesheet_link_tag 'utilities', nopush: false %>
```

Reply:
Whoops, thanks, that's embarrassing! Good thing the tests caught it as well :sweat_smile: